Lib/pickletools.py

```diff
@@ -2539,8 +2539,6 @@ def dis(pickle, out=None, memo=None, indentlevel=4, annotate=0):
         stack.extend(after)
 
     print("highest protocol among opcodes =", maxproto, file=out)
-    if stack:
-        raise ValueError("stack not empty after STOP: %r" % stack)
```

Comment:
Note that exception is raised after printing all output. You do not lose any information. You can catch and ignore this exception.

Reply:
Okay, so we're not worried about this then?

Reply:
> You can catch and ignore this exception.

It's not easy when using pickletools CLI.

Reply:
When using CLI you get some output. Whether it a Python traceback or a single line, -- it does not matter much, since such case is exceptional. The CLI output is not formalized, anything is fine.

We could add a mechanism for outputting non-critical errors, but since this particular error is raised after finishing all output, it would not make much difference.